packages/mobile/src/components/core/Hyperlink.tsx

```diff
@@ -68,47 +68,55 @@ export const Hyperlink = (props: HyperlinkProps) => {
    * android doesn't return the correct layout for nested text elements
    * */
   useEffect(() => {
```

Comment:
would useLayoutEffect help? prob not cause you actually want it running after the first paint?